### DISCUSSION TITLE
Dashboard Variables: support for variables in datasource regex

### DISCUSSION THREAD

**Discussion by matthiaslee** [grafana/grafana#72963] [Dashboards, Panels & Visualizations]
We have a setup where we have a number of datasources separated out by region, ideally after selecting the region in a separate variable, we could then use the `$region` to only show the relevant datasources for that region.

like so:
![image](https://github.com/grafana/grafana/assets/1522304/b594eca6-0cac-4266-8438-388dbe720144)

note: `$region` in this case does not get templated and confuses the regex.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82546. Feel free to continue the discussion around this there. Thank you!
